klusta_pipeline/openephys_to_kwd.py

```diff
@@ -0,0 +1,217 @@
+# Converts open ephys format files to kwd files
+import os
+import glob
+import numpy as np
+import h5py as h5
+
+# OpenEphys constants
+NUM_HEADER_BYTES = 1024
+SAMPLES_PER_RECORD = 1024
+BYTES_PER_SAMPLE = 2
+RECORD_SIZE = 4 + 8 + SAMPLES_PER_RECORD*BYTES_PER_SAMPLE + 10
+RECORD_RECORDING_OFFSET = 8+2
+RECORD_TIMESTAMP_OFFSET = 0
+
+
+def find_continuous_files(data_dir):
+    return glob.glob(os.path.join(data_dir, '*.continuous'))
+
+def find_events_files(data_dir):
+    return glob.glob(os.path.join(data_dir, '*.events'))
+
+def get_n_channels(continuous_files):
+    return len(continuous_files)
+
+def initialize_kwd_file(data_dir, experiment_name):
```

Comment:
have you looked at ```from klusta_pipeline.dataio import save_info, save_recording, save_chanlist, save_probe, save_parameters```
there might be a bunch of code you can reuse in there
Also, are we going to have averaging options here? as in klusta_pipeline.utils? options to ignore dead channels?